diff --git a/node_modules/@react-native-community/cli-platform-apple/build/commands/runCommand/runOnSimulator.js b/node_modules/@react-native-community/cli-platform-apple/build/commands/runCommand/runOnSimulator.js
index 4581437..02f10ef 100644
--- a/node_modules/@react-native-community/cli-platform-apple/build/commands/runCommand/runOnSimulator.js
+++ b/node_modules/@react-native-community/cli-platform-apple/build/commands/runCommand/runOnSimulator.js
@@ -42,7 +42,18 @@ async function runOnSimulator(xcodeProject, platform, mode, scheme, args, simula
   const activeDeveloperDir = _child_process().default.execFileSync('xcode-select', ['-p'], {
     encoding: 'utf8'
   }).trim();
-  _child_process().default.execFileSync('open', [`${activeDeveloperDir}/Applications/Simulator.app`, '--args', '-CurrentDeviceUDID', simulator.udid]);
+  try {
+    const fs = require('fs');
+    const simAppPath = `${activeDeveloperDir}/Applications/Simulator.app`;
+    if (fs.existsSync(simAppPath)) {
+      _child_process().default.execFileSync('open', [simAppPath, '--args', '-CurrentDeviceUDID', simulator.udid]);
+    } else {
+      const deviceHub = '/Applications/Xcode.app/Contents/Applications/DeviceHub.app';
+      if (fs.existsSync(deviceHub)) {
+        _child_process().default.execFileSync('open', ['-a', deviceHub]);
+      }
+    }
+  } catch (_e) {}
   if (simulator.state !== 'Booted') {
     bootSimulator(simulator);
   }
